ERESOLVE error during npm install

```
PS C:\Users\user\bgutil-ytdlp-pot-provider\server> npm install
npm error code ERESOLVE
npm error ERESOLVE could not resolve
npm error
npm error While resolving: bgutil-ytdlp-pot-provider@1.1.0
npm error Found: dev @commander-js/extra-typings@"github:commander-js/extra-typings.git#develop" from the root project
npm error
npm error Could not resolve dependency:
npm error dev @commander-js/extra-typings@"github:commander-js/extra-typings.git#develop" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
npm error
npm error
npm error For a full report see:
npm error C:\Users\user\AppData\Local\npm-cache\_logs\2025-08-06T12_51_51_758Z-eresolve-report.txt
npm error A complete log of this run can be found in: C:\Users\user\AppData\Local\npm-cache\_logs\2025-08-06T12_51_51_758Z-debug-0.log
```

I've tried both `--legacy-peer-deps` and `--force` but both result in the following error when I run `node build/main.js`:

```
node:internal/modules/cjs/loader:1413
  throw err;
  ^

Error: Cannot find module 'C:\Users\user\bgutil-ytdlp-pot-provider\server\build\main.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1410:15)
    at defaultResolveImpl (node:internal/modules/cjs/loader:1051:19)
    at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1056:22)
    at Module._load (node:internal/modules/cjs/loader:1219:37)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:238:24)
    at Module.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:154:5)
    at node:internal/main/run_main_module:33:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v24.5.0
```

--- a/server/package.json
+++ b/server/package.json
@@ -30,7 +30,7 @@
         "youtubei.js": "^15.0.1"
     },
     "devDependencies": {
-        "@commander-js/extra-typings": "github:commander-js/extra-typings.git#develop",
+        "@commander-js/extra-typings": "^14.0.0",
         "@eslint/js": "^9.32.0",
         "@types/body-parser": "^1.19.6",
         "@types/express": "^5.0.3",

